app/models/post.rb

```diff
@@ -2,4 +2,34 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+
+  after_create :increment_user_posts_counter
```

Comment:
- Please use the `after_save` method to call the `increment_function`, as we only want to increment the attribute only when the data is saved in the database, please fix this 👍